### DISCUSSION TITLE
default panel when deploying to production

### DISCUSSION THREAD

**Discussion by peterfoers** [filamentphp/filament#11882] [Ideas]
When deploying my app with filament admin panel, I received the error `No default Filament panel is set. You may do this with the default() method inside a Filament provider's panel() configuration. Error Type: Filament\Exceptions\NoDefaultPanelSetException` when trying to access `/admin`.

This was not an issue on local development nor my staging site, so I would have expected it to just display the default dashboard, like it did on these sites.

The solution to the problem was to set `->default(true)` in `AdminPanelProvider.php`. I could not find any reference to that in the documentation, nor does the error message point to that. Furthermore, the error message seems to suggest that one is to set an actual default page a la `->default('dashboard')`.

Suggestions:
- Mention setting default(true) in the error message
- Document the default() function on [the panel doc pages](https://filamentphp.com/docs/3.x/panels/configuration)
- If this makes sense: Either default the parameter to `true` or create the `AdminPanelProvider` with `default(true)`

**Comment by ryanmortier** [comment]
When you first install Filament it adds `->default()` to your first panel but yes I think more documentation around this would be good.
